background.md

```diff
@@ -0,0 +1,95 @@
+# Background
+
+## Motivating examples
```

Comment:
"Motivating Examples" can be in background.md, but I think we will need to add a short "Motivation" section to proposal.md in paragraph form.

Reply:
Why? I’m trying to keep the proposal as concise as possible. What would the shorter Motivation section say?